listenbrainz/webserver/static/js/src/recommendations/RecommendationCard.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from "react";
+import { get as _get } from "lodash";
+import MediaQuery from "react-responsive";
+import {
+  faAngry,
+  faSadCry,
+  faMeh,
+  faSmileBeam,
+  faLaughBeam,
+} from "@fortawesome/free-solid-svg-icons";
+
+import { getArtistLink, getTrackLink } from "../utils";
+import Card from "../components/Card";
+import RecommendationControl from "./RecommendationControl";
+
+export type RecommendationCardProps = {
+  recommendation: Recommendation;
+  playRecommendation: (recommendation: Recommendation) => void;
+  className?: string;
+  isCurrentUser: Boolean;
+  currentUser?: ListenBrainzUser;
+  newAlert: (
+    alertType: AlertType,
+    title: string,
+    message: string | JSX.Element
+  ) => void;
+};
+
+export default class RecommendationCard extends React.Component<
+  RecommendationCardProps
+> {
+  playRecommendation: (recommendation: Recommendation) => void;
+
+  constructor(props: RecommendationCardProps) {
+    super(props);
+
+    this.playRecommendation = props.playRecommendation.bind(
+      this,
+      props.recommendation
+    );
+  }
+
+  render() {
+    const { recommendation, className } = this.props;
+
+    return (
+      <Card
+        onDoubleClick={this.playRecommendation}
+        className={`recommendation-card row ${className}`}
+      >
+        <div className="col-xs-9">
+          <MediaQuery minWidth={768}>
+            <div className="col-xs-9">
+              <div className="track-details">
+                <p title={recommendation.track_metadata.track_name}>
+                  {getTrackLink(recommendation)}
+                </p>
+                <p>
+                  <small
+                    className="text-muted"
+                    title={recommendation.track_metadata.artist_name}
+                  >
+                    {getArtistLink(recommendation)}
+                  </small>
+                </p>
+              </div>
+            </div>
+          </MediaQuery>
+          <MediaQuery maxWidth={767}>
+            <div className="col-xs-12">
+              <div className="track-details">
+                <p title={recommendation.track_metadata.track_name}>
+                  {getTrackLink(recommendation)}
+                </p>
+                <p>
+                  <small
+                    className="text-muted"
+                    title={recommendation.track_metadata.artist_name}
+                  >
+                    {getArtistLink(recommendation)}
+                  </small>
+                </p>
+              </div>
+            </div>
+          </MediaQuery>
```

Comment:
```suggestion
            <div className="col-xs-12">
              <div className="track-details">
                <p title={recommendation.track_metadata.track_name}>
                  {getTrackLink(recommendation)}
                </p>
                <p>
                  <small
                    className="text-muted"
                    title={recommendation.track_metadata.artist_name}
                  >
                    {getArtistLink(recommendation)}
                  </small>
                </p>
              </div>
            </div>
```
We dont need Media queries as the layout is same for both Mobile and Desktop view